corehq/apps/app_manager/dbaccessors.py

```diff
@@ -3,8 +3,7 @@
 from collections import namedtuple
 from itertools import chain
 
-from couchdbkit.exceptions import DocTypeError
-from couchdbkit.resource import ResourceNotFound
+from couchdbkit.exceptions import DocTypeError, ResourceNotFound
```

Comment:
I think some of these could get merged ~now if you want to keep the final changeset small